### PR TITLE
feat: new Irish 1k language

### PR DESCRIPTION
**Pull request by EoinSMC**
### Description

Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/6107421846

File: frontend/static/languages/irish_1k.json [hunk 1]
```diff
@@ -0,0 +1,1023 @@
+{
+  "name": "irish_",
```
Comment (Shuja-Mahmood) [inline frontend/static/languages/irish_1k.json:2]:
```suggestion
  "name": "irish_1k",
```

File: frontend/static/languages/irish_1k.json [hunk 2]
```diff
@@ -0,0 +1,1023 @@
...
+    "féach",
+    "mise",
+    "bainisteoir",
+    "istigh",
+    "amuigh",
+    "ceantar",
+    "croí",
+    "glaoch",
+    "sásta",
+    "aimsiú",
```
Comment (Shuja-Mahmood) [inline frontend/static/languages/irish_1k.json:1021]:
```suggestion
    "aimsiú"
```
`ci-assets` failing due to this

File: frontend/static/languages/_groups.json [hunk 3]
```diff
@@ -171,7 +171,10 @@
   },
   {
     "name": "irish",
-    "languages": ["irish"]
+    "languages": [
+                  "irish",
+                  "irish_1k"
+    ]
```
Comment (Shuja-Mahmood) [inline frontend/static/languages/_groups.json:177]:
```suggestion
    "languages": [
      "irish",
      "irish_1k"
    ]
```
Let's match the indent with the rest of the file.

**Review by Shuja-Mahmood — CHANGES_REQUESTED** [review]
I've reviewed your code and left a few comments to help out.

**Pull request closed without merging**